src/main/java/de/muenchen/isi/api/dto/abfrageInBearbeitungSachbearbeitung/AbfragevarianteBauleitplanverfahrenSachbearbeitungInBearbeitungSachbearbeitungDto.java

```diff
@@ -26,6 +29,10 @@ public class AbfragevarianteBauleitplanverfahrenSachbearbeitungInBearbeitungSach
     @NotNull
     private SobonOrientierungswertJahr sobonOrientierungswertJahr;
 
+    private Boolean isASobonBerechnung;
+
```

Comment:
Evtl die beiden Variablen in einer Klasse auslagern welche dann bei den Entitäöten als Embedabble definiert sind. Diese Klasse dann anstelle der beiden Attribute in die Dto/Models/Entitäten einfügen.

Somit braucht nur eine Validierungslogik für die neue Klasse geschrieben werden und die einzelnen Validatoren und Validieirungsannotationen je Abfragetyp können entfallen.